VG Music Studio/Core/GBA/MP2K/Player.cs

```diff
@@ -86,7 +86,11 @@ private void SetTicks()
                     SongEvent e = evs.Single(ev => ev.Offset == track.DataOffset);
                     if (track.CallStackDepth == 0 && e.Ticks.Count > 0)
                     {
-                        break;
+                        // HACK: this check ensure that we can't get into an infinite jump
```

Comment:
You'll also need to remove these hacks if the jump offset hack is removed

Reply:
done